public/admin/config.yml

```diff
@@ -737,11 +737,48 @@ collections:
             widget: "string"
             required: false
             hint: This is the title for the Events section header. For example, "Main Stage Events." Defaults to 'Events.'
-          - name: "embed-link"
-            label: "Luma Embed URL"
-            widget: "string"
-            required: false
-            hint: Paste the embed URL for your Luma event here to display the events directly on the page. Include relevant tags when appropriate, e.g., https://lu.ma/embed/calendar/cal-nlDvL4B7Ko1swF0/events?lt=light&tag=FIL%20Bangkok%202024.
+          - name: "events"
+            label: "Events"
+            widget: "list"
+            required: true
+            fields:
+              - name: "title"
+                label: "Title"
+                widget: "string"
+              - name: "start-date"
+                label: "Start Date"
+                widget: "datetime"
+                hint: "First day of event"
+                picker_utc: true
+              - name: "end-date"
+                label: "End Date"
+                widget: "datetime"
+                required: false
+                hint: "Last day of event"
+                picker_utc: true
+              - name: "location"
+                label: "Location"
+                widget: "string"
+              - name: "description"
+                label: "Description"
+                widget: "string"
+                required: false
+              - name: "external-link"
+                label: "External Link URL"
+                widget: "object"
+                required: false
+                collapsed: true
+                fields:
+                  - name: "url"
+                    label: "URL"
+                    widget: "string"
+                    required: false
+                  - name: "text"
+                    label: "Text"
+                    widget: "string"
+                    required: false
+                    default: "View Event Details"
+                    hint: "Text for the external link CTA. Please use proper capitalization. Defaults to 'View Event Details.'"
```

Comment:
Curious why we would need this to be customisable? Do we want different CTAs on the cards? If yes, could we handle the capitalisation part on our side?